uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-record-actions.html

```diff
@@ -1,14 +1,28 @@
-<div class="btn-group" uib-dropdown>
-  <button type="button" class="btn {{vm.btnClass}} dropdown-toggle" uib-dropdown-toggle>
-    <span class="visible-sm visible-md visible-lg"><i class="fa fa-plus-circle" aria-hidden="true"></i> Record New Absence</span>
-    <span class="hidden-sm hidden-md hidden-lg"><i class="fa fa-plus" aria-hidden="true"></i> New</span>
+<div class="clearfix">
+  <div class="leave-request-record-actions__button btn-group" uib-dropdown>
+    <button type="button" class="btn btn-primary btn-lg dropdown-toggle" uib-dropdown-toggle>
+      <span class="hidden-xs">
+        <span class="btn-icon"><i class="fa fa-plus-circle"></i></span> Record New Absence
+      </span>
+      <span class="visible-xs">
+        <span class="btn-icon"><i class="fa fa-plus-circle"></i></span> New Absence
+      </span>
+    </button>
+    <ul class="dropdown-menu dropdown-menu-right" role="menu" uib-dropdown-menu>
+      <li ng-repeat="option in vm.leaveRequestOptions">
+        <a href
+          class="leave-request-record-actions__new-{{option.type}}"
+          ng-click="vm.openLeavePopup(null, option.type, vm.selectedContactId, vm.isSelfRecord)">
+          <span class="btn-icon"><i class="fa fa-{{option.icon}}"></i></span> {{option.label}}
+        </a>
+      </li>
+    </ul>
+  </div>
+  <button
+    class="leave-request-record-actions__button leave-request-record-actions__new-toil
```

Comment:
`leave-request-record-actions__new-toil` doesn't really serve any purpose here, the `leave-request-record-actions__button` class is sufficient

Reply:
BackstopJS.

Reply:
please point me to where this class is used

Reply:
https://github.com/civicrm/civihr/pull/2536/files#diff-d0f17a9ac26005f4074cebd80986836aR117 - this class allows to remove "unstable" (as per my opinion) Backstop function. It now allows to not care about the amount and order of buttons.